index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <main id = "main">
+     <h1 id = "title">
+       Dr. Norman Borlaug
+     </h1>
+     <div id= "img-div">
+       <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">
```

Comment:
I noticed that you did not put anything in the alt attribute of your image. This attribute is very important because it will give alternative information or description about the image if the image does not load due to a slow connection or src (source) error. Kindly, write something in the 'alt' attribute and please make sure that it describes the image.